deployment.md

```diff
@@ -0,0 +1,33 @@
+# VFOX 0.1 Deployment Guide
+
+### Purpose
+ Contain information about the installation process of backend as JAVA and frontend as Angular.
+### Prerequisites
+```
+Java 1.8+
```

Comment:
We need to add commands to install all these.

Reply:
Added install command for java 8 and nginx.
Removed Mysql and docker, as it is already been taken care in README.md file.